docs/tutorials/platform_beginner/7_customize_login_and_register_forms.md

```diff
@@ -0,0 +1,146 @@
+# Step 7 - Customize the User forms
```

Comment:
I suggest `Customizing`, as in the remaining page titles

Also, `user` lowercase, like in the rest of the text